recipes/quantlib/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "QuantLib" %}
+{% set version = "1.26" %}
+{% set sha256 = "04fe6cc1a3eb7776020093f550d4da89062586cc15d73e92babdf4505e3673e9" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/lballabio/QuantLib/releases/download/{{ name }}-v{{ version }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: true  # [win and vc<14]
+  run_exports:
+    - boost-cpp
```

Comment:
```
run_exports:
  - pin_compatible('quantlib', max_pin=x)
```
Surely we mean to export a dependency on quantlib and not on boost. If we compile against quantlib, then we need quantlib?

Reply:
Yeah this is probably copypasta from the underlying quantlib c++